docs/dm/zh_CN/maintenance/dm-ansible.md

```diff
@@ -8,7 +8,7 @@ Ansible 是一款自动化运维工具，DM-ansible 是 PingCAP 基于 Ansible P
 本部署工具可以通过配置文件设置集群拓扑，完成以下各项运维工作：
 
 - 初始化部署机器操作系统参数
-- 部署 DM 集群组件（包括 DM-master、DM-worker、dmctl）
+- 部署 DM 集群组件（包括 dm-master、dm-worker、dmctl）
```

Comment:
copied from DM?
they become lower case now.

Reply:
use `DM-worker` and `DM-master`